Oculo/HealthkitManager.swift

```diff
@@ -129,17 +129,20 @@ class HealthKitManager {
         }
     }
 
-    public func retrieveWalkingStepLength(period : Period, completion : @escaping (Double) -> Void) { // 건강앱의 보폭데이터를 받아오는 메소드
-        guard let walkingStepLengthType = HKSampleType.quantityType(forIdentifier: .walkingStepLength) else { return } //보폭데이터를 받아올것이라는타입 지정
+    /// 건강 앱의 보폭 데이터를 받아오는 메소드
+    public func retrieveWalkingStepLength(period : Period, completion : @escaping (Double) -> Void) {
```

Comment:
```suggestion
    public func retrieveWalkingStepLength(period: Period, completion: @escaping (Double) -> Void) {
```
후에 띄어쓰기 수정을 하면 좋을 것 같습니다!